File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.length()!=t.length()) return false;
        vector<int>v(26,0);
        for(auto it: s) v[it-'a']++;
        for(auto it: t){
            if(v[it-'a']<=0) return false;
            v[it-'a']--;
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int n = s.length(),m=t.length();
        if(n!=m) return false;
        int []arr = new int[26];
        Arrays.fill(arr,0);
        for(char c: s.toCharArray()) {
            arr[c-'a']++;
        }
        for(char c: t.toCharArray()) {
            arr[c-'a']--;
        }
        
        for(int a: arr) {
            if(a!=0) return false;
        }
        return true;
    }
}